lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
 
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
+  hash = {}
+  results = []
+
+  string.each_char do |char|
+    if hash[char] == nil
+      hash[char]== 0
+    else
+      hash[char] += 1
+    end
+  end
+  
+  hash.each_char do |char|
```

Comment:
Hashes don't have a `.each_char` method.  Instead you can use:

```suggestion
hash.each do |char, count| 
```